docs/platform/howto/create_new_service_user.md

```diff
@@ -2,36 +2,39 @@
 title: Create service users
 ---
 
-import Users from "@site/static/images/icons/people.svg";
-
-Service users are users that only exist in the scope of the corresponding Aiven service.
-They are unique to this service and not shared with any other services,
-and can be granted restricted permissions compared to the default `avnadmin`
-user.
-
-You can add service users for all the Aiven services except Aiven for Apache Flink®
-and Aiven for Grafana®.
-
-:::note
+Service users are users that only exist in the scope of the
+corresponding Aiven service. They are unique to this service and not
+shared with any other services, and can be granted restricted
+permissions compared to the default `avnadmin` user. You can add service
+users for all the Aiven services, with the exception of Aiven for Apache
+Flink® and Aiven for Grafana®.
+
+:::warning
 By default, the maximum amount of users allowed on a service is 50.
 
 To increase the maximum number of users allowed for a service,
-[create a support ticket](/docs/platform/howto/project-support-center) to request an increase.
+[create a support ticket](/docs/platform/howto/support) to request an increase.
 :::
 
+## Create a service user
+
 1.  Log in to [Aiven Console](https://console.aiven.io/).
 1.  On the **Services** page, select your service.
-1.  From the sidebar, click <Users className="icon"/> **Users**.
-1.  Click **Add service user**:
```

Comment:
And we can bring these 2 back, at least the icon, but for this file, I feel like this is a merge conflict that went south. I would suggest considering bringing back the original version from line 24, where I shortened things a tad earlier.